docs/endpoints.rst

```diff
@@ -237,6 +237,8 @@ What is the density of restaurants with wheelchair access in Heidelberg?
    :query types2: Deprecated! use **filter2**
    :query values2: Deprecated! see **filter2**
    
+.. note:: The result of a **ratio request** may contain the value **"NaN"**, for example when dividing `0` by `0`. 
```

Comment:
IMHO the wording here is not great: a) The use of "_for example_" suggests that there are other situations where this could also occur, but AFAIK there aren't. b) The verb "_dividing_" suggests that this is something the user has done or can control; it could be made clearer that this is a result of a calculation where such a division occurred.

> .. note:: The result of a **ratio request** may contain the value **"NaN"**, when the ratio calculation involves a division of zero by zero.

Reply:
Good point, that's not in general the only operation where `NaN` could be the result, but as we do a division here for the ratio, it is in fact in our case. It's a bit unlucky that your suggestion did not come earlier :D but feel free to make a quick PR for that change if you'd like to.

Reply:
:arrow_right:  https://github.com/GIScience/ohsome-api/pull/172